homeassistant/components/panel_custom.py

```diff
@@ -42,6 +43,7 @@
         vol.Optional(CONF_CONFIG): dict,
         vol.Optional(CONF_WEBCOMPONENT_PATH): cv.isfile,
         vol.Optional(CONF_JS_URL): cv.string,
+        vol.Optional(CONF_JS_IS_MODULE): cv.boolean,
```

Comment:
I am fine with the functionality but I am not a big fan of this config option.

I would prefer if you add a new option ` CONF_MODULE_URL` instead.

Reply:
The reason I wrote it like that is just that you need to check for a three way option conflict between html_url, js_url and module_url.
```python
elif (js_url and html_url) or (js_url and module_url) or (module_url and html_url):
        raise ValueError('Pass in only one of JS url, module url or HTML url.')
```

which looked messy to me.
I'll change the code around with module_url in mind.

Reply:
You can leverage `vol.Exclusive`

Reply:
Thanks I'll do that